Amazon.Ion.ObjectMapper/IonObjectSerializer.cs

```diff
@@ -75,8 +103,46 @@ public override void Serialize(IIonWriter writer, object item)
         {
             options.TypeAnnotator.Apply(options, writer, targetType);
             writer.StepIn(IonType.Struct);
+
+            var serializedIonFields = new HashSet<string>();
+            
+            // Serialize the values returned from IonPropertyGetter annotated getter methods.
+            foreach (var method in targetType.GetMethods())
```

Comment:
Collapse 110 and 112 and pull to a private method which extracts the appropriately annotated get methods.

Reply:
Done.